src/pybamm/citations.py

```diff
@@ -138,7 +127,7 @@ def register(self, key):
                     self._add_citation_tag(key, entry=caller)
                     # Don't add citation tags if the citation is registered manually
                 except KeyError:  # pragma: no cover
-                    pass
+                    self._module_import_error = True
```

Comment:
This was a mistake, fixing it
```suggestion
                    pass
```